DOCUMENTATION.md

```diff
@@ -49,19 +49,19 @@ client.close()
 The callback functions are functions which are executed when a trigger event happens.
 
 #### set_uplink_callback
-Set the callback function, to be called when an uplink message is received.
+Set a callback function, to be called when an uplink message is received. It's possible to add more than one callback function.
```

Comment:
I'd formulate it more as *Add a callback function to be called when...* rather than the current *Set a... It's possible to...*.